tests/integration/components/public/event-map-test.js

```diff
@@ -0,0 +1,35 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('g-map', 'Integration | Component | g map', {
```

Comment:
And your test needs to be for `public/event-map` component. Not `g-map` component. `g-map` is already well tested by the library developer.